tests/foreman/rhai/test_rhai_client.py

```diff
@@ -14,70 +14,73 @@
 
 :Upstream: No
 """
+import logging
+
 import pytest
-from fauxfactory import gen_string
+from fauxfactory import gen_alpha
 from nailgun import entities
 
 from robottelo import manifests
 from robottelo.api.utils import upload_manifest
 from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
-from robottelo.constants import DISTRO_RHEL6
+from robottelo.constants import DISTRO_DEFAULT
 from robottelo.decorators import skip_if_not_set
-from robottelo.test import TestCase
 from robottelo.vm import VirtualMachine
 
 
+logger = logging.getLogger('robottelo')
+
+
+@pytest.fixture(scope='module')
+def org():
+    """Create a new organization with name prefix 'insights_'"""
+    return entities.Organization(name=gen_alpha(15, start='insights_')).create()
+
+
+@pytest.fixture(scope='module')
+def manifest(org):
+    """Upload manifest to organization."""
+    with manifests.clone() as manifest:
+        upload_manifest(org.id, manifest.content)
```

Comment:
could change this to something like `manifest_org` and return the org

Reply:
Done.